.travis.yml

```diff
@@ -2,8 +2,6 @@ dist: xenial
 language: python
 cache: pip
 python:
-  - "2.7"
-  - "pypy"
   - "3.4"
```

Comment:
Probably want to just require 3.5 or better, as I've introduced some code and programming patterns that require it. (other stuff should be cleaned up).